### DISCUSSION TITLE
Possible solution to people having the same problem as in #5631

### DISCUSSION THREAD

**Discussion by omgMath** [strapi/strapi#8687] [Legacy Topics]
Some people (including me) had the problem that the couldn't create new collection types after upgrading to 3.3.3 (see https://github.com/strapi/strapi/issues/5631).

This can be resolved by running

`strapi build`

and I think it would be helpful to add this information within the issue.

Buuut, since the issue is locked, I cannot add it. Not sure if this is the right channel to request that, but could you, @derrickmehaffy add that info?

Thanks!

**Comment by derrickmehaffy** [comment]
I've added your comment to the issue, but for clarification:

> Please note with regards to updating versions, see this part of our update/migration documents:
> https://strapi.io/documentation/v3.x/migration-guide/
> https://strapi.io/documentation/v3.x/guides/update-version.html#rebuild-your-administration-panel

> Every version update, even minor you should be rebuilding your Admin panel.

**Reply by omgMath** [reply]
Thank you. (But I'm not baermathias :sweat_smile: as stated in the issue)

**Reply by derrickmehaffy** [reply]
Ah shoot sorry
